sky/backends/cloud_vm_ray_backend.py

```diff
@@ -948,7 +951,8 @@ def _retry_region_zones(self,
             # This sets the status to INIT (even for a normal, UP cluster).
             global_user_state.add_or_update_cluster(cluster_name,
                                                     cluster_handle=handle,
-                                                    ready=False)
+                                                    ready=False,
+                                                    task=task)
```

Comment:
nit: I would prefer to only pass the `requested_resources` to the function to avoid the `global_user_state` functions knowing the concept of `sky.Task`.